pkg/cmd/repo/create/create.go

```diff
@@ -404,6 +411,114 @@ func createFromScratch(opts *CreateOptions) error {
 		} else if err := cloneWithRetry(opts, remoteURL, templateRepoMainBranch); err != nil {
 			return err
 		}
+
+	}
+
+	return nil
+}
+
+// create new repo on remote host from template repo
+func createFromTemplate(opts *CreateOptions) error {
+	// this method is only for interactive mode
+	if !opts.Interactive {
```

Comment:
You are right. I fixed the code to return an error.

Reply:
Thanks. I'd also be fine removing this check altogether. This would be a programmer error and be so obviously broken that it wouldn't really get very far anyway. I looked around the codebase and we don't typically do these kind of checks. It would likely result in them appearing in many places around the codebase, which would be kind of noisy.